server/src/test/java/org/elasticsearch/common/geo/GeometryTreeTests.java

```diff
@@ -241,15 +241,20 @@ public void testRandomGeometryIntersection() throws IOException {
 
         Geometry geometry = randomGeometryTreeGeometry();
         GeoShapeIndexer indexer = new GeoShapeIndexer(true, "test");
-        geometry = indexer.prepareForIndexing(geometry);
+        Geometry preparedGeometry = indexer.prepareForIndexing(geometry);
+
+        // TODO: support multi-polygons
+        if (ShapeType.POLYGON == geometry.type() && ShapeType.MULTIPOLYGON == preparedGeometry.type()) {
```

Comment:
I think assumeFalse would work better here showing us if the test actually ran or not.

Reply:
oh right. I forgot about that method. usually only ever thought of it in the context of different Operating System stuff